package punto4;

public class Excepcion {
	
	public static void main(String[] args) {
		
		try {
			  // Código que podría producir una excepción
			  int x = y + 1;
			} catch (error) {
			  // Manejo del error
			  System.out.println("Ha ocurrido un error: " + error.message);
			}

	}

}
